--- libs3-287e4bee6fd430ffb52604049de80a27a77ff6b4/GNUmakefile.orig	2020-05-22 07:45:08.338698313 +0200
+++ libs3-287e4bee6fd430ffb52604049de80a27a77ff6b4/GNUmakefile	2020-05-22 07:47:51.327815325 +0200
@@ -151,7 +151,7 @@
           -D_ISOC99_SOURCE \
           -D_POSIX_C_SOURCE=200112L
 
-LDFLAGS = $(CURL_LIBS) $(LIBXML2_LIBS) $(OPENSSL_LIBS) -lpthread
+LDFLAGS = $(LDOPTS) $(CURL_LIBS) $(LIBXML2_LIBS) $(OPENSSL_LIBS) -lpthread
 
 STRIP ?= strip
 INSTALL := install --strip-program=$(STRIP)
