--- libs3-287e4bee6fd430ffb52604049de80a27a77ff6b4/GNUmakefile~	2021-10-06 22:10:24.000000000 +0200
+++ libs3-287e4bee6fd430ffb52604049de80a27a77ff6b4/GNUmakefile	2021-10-06 22:11:31.883203755 +0200
@@ -142,7 +142,7 @@
     endif
 endif
 
-CFLAGS += -Wall -Werror -Wshadow -Wextra -Iinc \
+CFLAGS += -Wall -Wshadow -Wextra -Iinc \
           $(CURL_CFLAGS) $(LIBXML2_CFLAGS) \
           -DLIBS3_VER_MAJOR=\"$(LIBS3_VER_MAJOR)\" \
           -DLIBS3_VER_MINOR=\"$(LIBS3_VER_MINOR)\" \
